kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/filter/SpecClassFilter.kt

```diff
@@ -0,0 +1,7 @@
+package io.kotest.core.filter
+
+interface SpecClassFilter: Filter {
+
+}
+
+annotation class SpecClassExecutionFilter(val filter: String)
```

Comment:
Or we can use not the String - something from new Interface (and the users can implement this for their needs as they want)